apps/block_scout_web/lib/block_scout_web/templates/layout/app.html.eex

```diff
@@ -33,7 +33,7 @@
     <link rel="icon" type="image/png" sizes="32x32" href="<%= static_path(@conn, "/images/favicon-32x32.png") %>">
     <link rel="icon" type="image/png" sizes="16x16" href="<%= static_path(@conn, "/images/favicon-16x16.png") %>">
     <link rel="manifest" href="<%= static_path(@conn, "/manifest.webmanifest") %>">
-    <link rel="mask-icon" href="<%= static_path(@conn, "/safari-pinned-tab.svg") %>" color="#5bbad5">
+    <link rel="mask-icon" href="<%= static_path(@conn, "/images/favicon-32x32.png") %>">
```

Comment:
The png file has to be renamed, or update the path here
```suggestion
    <link rel="mask-icon" href="<%= static_path(@conn, "/images/favicon_32x32.png") %>">
```

Reply:
done